src/map/script.cpp

```diff
@@ -21594,7 +21600,7 @@ BUILDIN_FUNC(instance_warpall)
 	}
 
 	for(const auto &it : idata->map)
-		map_foreachinmap(buildin_instance_warpall_sub, it.m, BL_PC, map_id2index(m), x, y, instance_id);
+		map_foreachinmap(buildin_instance_warpall_sub, it.m, BL_PC, map_id2index(m), x, y, instance_id, flag);
```

Comment:
we shouldn't pass a uint8 to a variadic function, it can cause some issues.
instead, pass it as a normal int and we can cast it in the `warpall_sub` function